modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/paths/TestrayBuild.path

```diff
@@ -50,6 +50,16 @@
 	<td>//tbody//tr[contains(@class,'table-row')]//td//span[contains(@title,'promoted')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SELECT_STACKS_PARAMETER_LABEL_FIELD</td>
+	<td>//div[@class='modal-body']/div/label[text()='${parameterLabel}']//following::option[text()='${stackoptions}']</td>
+	<td></td>
+</tr>
```

Comment:
Would you mind to change the xpath ...//label... instead of /div/label ?

Reply:
ok